### PR TITLE
fix: disable base64 inlining of assets during build

### PR DESCRIPTION
**Pull request by im-adithya**
## Before
Screenshot 2024-08-15 at 3 40 38 PM

## After
Screenshot 2024-08-15 at 3 40 10 PM
Screenshot 2024-08-15 at 3 56 21 PM

### CODE REVIEW

File: frontend/src/assets/images/node/lnd.png
Comment (rolznz) [inline frontend/src/assets/images/node/lnd.png:1]: Does this have a white BG? it might look bad in dark mode

Comment (im-adithya) [reply frontend/src/assets/images/node/lnd.png:1]: Ah true let me find a png

**Comment by rolznz** [comment]
I noticed a slight styling issue on the LND image which I fixed on the PR I closed. In SetupNode we can change to this:
``
<div className="h-6 w-6 flex items-center justify-center">
                    {item.icon}
                  </div>
```

which will make the LND image centered vertically.

File: frontend/src/assets/images/node/lnd.png
Comment (im-adithya) [reply frontend/src/assets/images/node/lnd.png:1]: Done!

**Comment by im-adithya** [comment]
I fixed the image by making it a square png like others, so we will not need this styling 👍

**Review by rolznz — APPROVED** [review]
utACK

**Pull request merged**